sandy/fy.py

```diff
@@ -357,6 +357,115 @@ def apply_qmatrix(self, zam, energy, decay_data):
         new_data.loc[mask, 'FY'] = fy_calc.values
         return self.__class__(new_data)
 
+    def gls_cov_update(self, zam, e, decay_data, Vy=None, kind='cumulative',
+                       threshold=None, index_columns=None):
+        """
+        IFY covariance matrix performed by GLS for a given energy and zam.
+
+        Parameters
+        ----------
+        zam : `int`
+            ZAM number of the material to which calculations are to be
+            applied.
+        e : `float`
+            Energy to which calculations are to be applied.
+        decay_data : `sandy.DecayData`
+            Radioactive nuclide data for several isotopes.
+        Vy : 2D iterable, optional
+            Extra Covariance matrix (MXM). The default is None, so the method
+            uses the variance of the CFY for a given zam and e.
```

Comment:
For the moment I would like Vy not to be an optional keywords. In theory the default option should be chain yields.